mmv1/third_party/terraform/services/bigquery/resource_bigquery_table.go

```diff
@@ -622,6 +622,13 @@ func ResourceBigQueryTable() *schema.Resource {
 							},
 						},
 
+						"json_extension": {
```

Comment:
off first glance there seems to be a missing test related to `json_extension`, be sure to include a test case since we tend to get failed VCR tests if no tests that include the new field are not found.

Reply:
<img width="1098" alt="Screenshot 2024-03-20 at 22 30 54" src="https://github.com/GoogleCloudPlatform/magic-modules/assets/33663870/578f7493-88c7-43fa-9aed-005aae27cdc9">
It is covered though

Reply:
My mistake!